themes/cypress/languages/pt-br.yml

```diff
@@ -39,6 +39,7 @@ sidebar:
     organizations: Organizações
     users: Usuários
     github-integration: Integração com GitHub
+    slack-integration: Slack Integration
```

Comment:
Do these get automatically updated? It looks like the GitHub integration fields are translated but these aren't.

Reply:
No, there is no automation on our translations other than copying over the English docs over to the other language when they don't exist. We've been accepting PRs from users if they want to translate things.